From 5c01278accf52cb4ef9f164ce3b5c0aca08c3f16 Mon Sep 17 00:00:00 2001
From: Takahiro Ueda <t.ueda.od@juntendo.ac.jp>
Date: Tue, 9 Apr 2024 21:09:35 +0900
Subject: [PATCH] docs: building documents without repository (#497)

Avoid typesetting errors even when there is no revision information from
the repository. The absence of revision information is indicated by
"???" appended to the version number shown on the title page of the
documents.
---
 doc/devref/devref.tex.in | 2 +-
 doc/manual/manual.tex.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/devref/devref.tex.in b/doc/devref/devref.tex.in
index 34f22150..f63f4603 100644
--- a/doc/devref/devref.tex.in
+++ b/doc/devref/devref.tex.in
@@ -2,7 +2,7 @@
 \def\formversiondesc{\repoversiondesc}
 \def\formdate{\repodate}
 %begin{latexonly}  % To avoid latex2html/latex2html#37
-\providecommand{\formversiondesc}{@VERSION@}
+\providecommand{\repoversiondesc}{@VERSION@???}
 \providecommand{\repodate}{\today}
 %end{latexonly}
 
diff --git a/doc/manual/manual.tex.in b/doc/manual/manual.tex.in
index 4aa1a69c..17b5bba8 100644
--- a/doc/manual/manual.tex.in
+++ b/doc/manual/manual.tex.in
@@ -4,7 +4,7 @@
 \def\formversiondesc{\repoversiondesc}
 \def\formdate{\repodate}
 %begin{latexonly}  % To avoid latex2html/latex2html#37
-\providecommand{\formversiondesc}{@VERSION@}
+\providecommand{\repoversiondesc}{@VERSION@???}
 \providecommand{\repodate}{\today}
 %end{latexonly}
 
